packages/prosemirror-resizable-view/package.json

```diff
@@ -37,7 +37,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.13.10",
-    "@remirror/core-helpers": "^1.0.0-next.60",
+    "@remirror/core": "1.0.0-next.60",
```

Comment:
Try not to do it. `prosemirror-resizable-view` is a package independent from the remirror. You can use this package in a non-remirror project. `@remirror/core-helper` is a simple dependency like `lodash` but `@remirror/core` is much heavier.